xdp-dump/tests/test-xdpdump.sh

```diff
@@ -313,6 +320,8 @@ test_snap()
 
 test_multi_pkt()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
+
```

Comment:
You should be able to make this test pass with the older kernel, guess only lines 325-326 will probably need to be conditional.